lib/features/digital_guide_view/presentation/widgets/digital_guide_features_section.dart

```diff
@@ -53,10 +54,25 @@ class DigitalGuideFeaturesSection extends StatelessWidget {
         title: context.localize.elevators,
         content: [LocalizationExpansionTileContent()],
       ),
-      (
-        title: context.localize.toilets,
-        content: [LocalizationExpansionTileContent()],
-      ),
+      if (digitalGuideResponseExtended.levels.fold(
+            0,
+            (levelValue, levelElement) =>
+                levelValue +
+                levelElement.regions.fold(
+                  0,
+                  (regionValue, regionElement) =>
+                      regionValue + regionElement.adaptedToiletsIndices.length,
+                ),
+          ) >
+          0)
+        (
```

Comment:
I think you can separate it to higher order fuctions, because readablity of this part of code is low.